node/src/service.rs

```diff
@@ -3,9 +3,10 @@
 use encointer_node_notee_runtime::{self, opaque::Block, RuntimeApi};
 use sc_client_api::BlockBackend;
 use sc_consensus_aura::{ImportQueueParams, SlotProportion, StartAuraParams};
+use sc_consensus_grandpa::SharedVoterState;
 pub use sc_executor::NativeElseWasmExecutor;
-use sc_finality_grandpa::SharedVoterState;
-use sc_keystore::LocalKeystore;
+use sc_executor::{HeapAllocStrategy, DEFAULT_HEAP_ALLOC_STRATEGY};
+//use sc_keystore::LocalKeystore;
```

Comment:
Can we remove this one? Btw. here I also do the diff with the node-template runtime when I do the upgrade: https://github.com/paritytech/substrate/tree/master/bin/node-template